build.gradle.kts

```diff
@@ -0,0 +1,161 @@
+        relocate("dev.vankka.simpleast", "github.scarsz.discordsrv.dependencies.simpleast")
+        relocate("org.bstats", "github.scarsz.discordsrv.dependencies.bstats")
+        relocate("org.apache.commons", "github.scarsz.discordsrv.dependencies.commons")
+        relocate("org.json.simple", "github.scarsz.discordsrv.dependencies.json.simple")
+        relocate("org.json", "github.scarsz.discordsrv.dependencies.json")
+        relocate("org.minidns", "github.scarsz.discordsrv.dependencies.minidns")
+        relocate("okhttp3", "github.scarsz.discordsrv.dependencies.jda")
+        relocate("okio", "github.scarsz.discordsrv.dependencies.okio")
+        relocate("gnu.trove", "github.scarsz.discordsrv.dependencies.trove")
+        relocate("com.fasterxml.jackson", "github.scarsz.discordsrv.dependencies.jackson")
+        relocate("com.google.common", "github.scarsz.discordsrv.dependencies.google.common")
+        relocate("com.google.errorprone", "github.scarsz.discordsrv.dependencies.google.errorprone")
+        relocate("com.google.gson", "github.scarsz.discordsrv.dependencies.google.gson")
+        relocate("com.google.j2objc", "github.scarsz.discordsrv.dependencies.google.j2objc")
+        relocate("com.github.kevinsawicki", "github.scarsz.discordsrv.dependencies.kevinsawicki")
+        relocate("com.github.zafarkhaja", "github.scarsz.discordsrv.dependencies.zafarkhaja")
+        relocate("alexh", "github.scarsz.discordsrv.dependencies.alexh")
+
+        // Merge META-INF/services files where needed
+        mergeServiceFiles()
+
+        // Exclude signatures, maven/ and proguard/ from META-INF
+        exclude("META-INF/*.SF")
+        exclude("META-INF/*.DSA")
+        exclude("META-INF/*.RSA")
+        exclude("META-INF/maven/**")
+        exclude("META-INF/proguard/**")
+    }
+}
+
+fun configureKotlin(kotlinCompile: Action<KotlinCompile>) {
+    tasks.withType<KotlinCompile>().configureEach(kotlinCompile)
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+    maven("https://hub.spigotmc.org/nexus/content/repositories/snapshots/")
+    maven("https://papermc.io/repo/repository/maven-public/")
+    maven("https://oss.sonatype.org/content/repositories/central")
+    maven("https://oss.sonatype.org/content/repositories/snapshots")
+    maven("https://nexus.scarsz.me/content/groups/public/") {
+        content {
+            includeGroup("github.scarsz")
+            includeGroup("me.scarsz")
+            includeGroup("me.minecraftauth")
+        }
+    }
+}
+
+dependencies {
+    compileOnly("com.destroystokyo.paper:paper-api:${minecraftVersion}-R0.1-SNAPSHOT") {
+        exclude("commons-lang") // we have our own
+    }
+    implementation("net.dv8tion:JDA:4.4.0_352.fix-2") {
+        exclude("opus-java") // we don't use voice features
+    }
+    implementation("github.scarsz:configuralize:1.3.2")
+    implementation("org.bstats:bstats-bukkit:2.2.1")
+
+    // logging
+    api("me.scarsz:jdaappender:1.0.3")
```

Comment:
Does this need to use the `api` configuration?

Reply:
Not really, no, can be downgraded. Left it in because other plugins might want to customize DiscordSRV's JDA appender config but I guess that's a very small use case